### ISSUE TITLE
EPUB A11y 1.0 xpaths missing opening quote

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#511]
In the conformance section, the current xpaths are like this:

> text() = http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa"

But there needs to be a quote to start the URLs.
